index.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Technical Documentation Page</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <main id="main-doc">
+      <nav id="navbar">
+        <header>Product Landing Page</header>
+        <div>
+          <a href="#introduction" class="nav-link">Introduction</a>
+          <a href="#What_you_should_already_know" class="nav-link"
+            >What you should already know</a
+          >
+          <a href="#JavaScript_and_Java" class="nav-link"
+            >JavaScript and Java</a
+          >
+          <a href="#Variables" class="nav-link">Variables</a>
+          <a href="#Function_declarations" class="nav-link"
+            >Function declarations</a
+          >
+        </div>
+      </nav>
+
+      <section class="main-section" id="introduction">
+        <header>introduction</header>
```

Comment:
- your <header> tag should wrap the toolbar or the nav not the sections title. use heading(h2,h3,h4,...)tags instead.
Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation.

Reply:
You should try to communicate to the student clearly why you are suggesting the change, for example when you say " You tag should wrap.......", it's a bit confusing because there are many tags in your code, better clarify what tag you talking about for a student to understand the change that you are suggesting.

Reply:
- your <header> tag should wrap the toolbar or the nav not the title of sections. you could use  the heading(h2,h3,h4,...) tags instead.

Typically the <header> tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation.

Reply:
- [x] [REQUIRED] Please when giving code review, it is just appropriate to phrase your comments politely, as suggestions instead of orders. For example:

* This is a polite suggestion: *"I think your codebase would be cleaner if you remove unused code block commented out. Kindly remove the above commented code blocks"*
* This reads as an order: *"Remove comments inside the code blocks to avoid chunks"* 

- [x] Also, your comment will be more useful if it includes an explanation, a "why", for each requested change. For example:

* This is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*
* This request lacks a "why": *Why don't you use an article tag here instead of a div?*

Reply:
- I think your header tag should wrap the toolbar or the nav not the title of sections. you could use the heading(h2,h3,h4,...) tags instead.

Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation.